hack/mkdocs/make-docs.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```

Comment:
The make-docs.sh script
* Reuses some of the front matter content from Gateway make docs
* Gets version from the branch
* Checks if the version should be marked as latest
* Makes the versioned docs by using the `mike` tool

Reply:
The `make docs` script that the release calls would run this script. People could also check out the branch they want to build content for and run `make docs`.